dkg/exchanger.go

```diff
@@ -30,18 +30,20 @@ import (
 // to get support from parsigex and parsigdb core components. This may subject to change when DKG
 // package has its own networking and database components.
 // Values of following constants should not change as it can break backwards compatibility.
+type sigType int
+
 const (
 	// dutyLock is responsible for lock hash signed partial signatures exchange and aggregation.
-	dutyLock core.DutyType = 101
+	sigLock sigType = 101
 	// dutyDepositData is responsible for deposit data signed partial signatures exchange and aggregation.
-	dutyDepositData core.DutyType = 102
```

Comment:
since we need to use the same duty type `DutySignature`, we need to differentiate between these two use-cases using the slot. We can look at refactoring `core.Duty` to a something like `core.Task` which has a `Type` and a list of labels...?

Reply:
Another option would be to register a factory with `parsigex` at runtime, then DKG can implement its own stuff

Reply:
I am more inclined towards second option